transaction/core/src/validation/error.rs

```diff
@@ -160,3 +163,11 @@ impl From<InputRuleError> for TransactionValidationError {
         Self::InputRule(src)
     }
 }
+
+impl From<TxOutConversionError> for TransactionValidationError {
+    fn from(src: TxOutConversionError) -> Self {
+        match src {
+            TxOutConversionError::UnknownMaskedAmountVersion => Self::UnknownMaskedAmountVersion,
```

Comment:
Any reason to do this versus include a `TxOutConversion(TxOutConversionError)` variant in the `TransactionValidationError` enum?